steering/elections/2020/jimmysong.md

```diff
@@ -0,0 +1,13 @@
+# Jimmy Song
+
+- GitHub: https://github.com/rootsongjc
+- Affiliation: developer advocate, Tetrate.io
+- Slack: @Jimmy Song
+- Twitter: @jimmysongio
+- LinkedIn: https://www.linkedin.com/in/jimmysongio/
+
+# Bio
+Jimmy is a developer advocate at Tetrate, CNCF ambassador, 
```

Comment:
Isn't Jimmy also one of the key maintainers for istio.io chinese website?

Reply:
He wrote the bio for me :) @rootsongjc anything to update?

Reply:
Yes, I am the initiator and key maintainer of the Istio Chinese website.